.github/ISSUE_TEMPLATE/design_call.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Community or working group call
+about: Bitcoin Design community call or design sessions
+title: '... Design Call #0'
+labels: 'call'
+assignees: ''
```

Comment:
Who is an assignee in the case of a call? The organizer?

Reply:
Currently it's just left as blank. Don't think it's useful to explicitly set unless the issue being posted is on the behalf of someone else.